src/main/resources/sql/fill-table-subcategory.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO `evo_exchange`.`subcategory` (`id`, `category_id`, `name`)
```

Comment:
We should definitely add Flyway to our project and move this logic to that framework...